qtcodes/fitters/lattice_decoder.py

```diff
@@ -494,7 +497,8 @@ def node_to_pos3D(node):
         # 3D network plot
         with plt.style.context(("ggplot")):
             fig = plt.figure(figsize=(20, 14))
-            ax = Axes3D(fig)
+            ax = Axes3D(fig, auto_add_to_figure=False)
```

Comment:
Why is this change necessary? (Is the original code being deprecated?)

Reply:
Yes.